src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -666,6 +666,12 @@ configurations:
             - type: xsltFileUpgrader
               template: crafter/studio/upgrade/4.0.x/config/permission-mappings-config/permission-mappings-config-v4.0.6.xslt
               commitDetails: Add content_search permissions for all users
+        - currentVersion: 4.0.6
+          nextVersion: 4.0.7
+          operations:
+            - type: xsltFileUpgrader
+              template: crafter/studio/upgrade/4.0.x/config/permission-mappings-config/permission-mappings-config-v4.0.7.xslt
+              commitDetails: Add log_monitor permission for developer role
```

Comment:
Not sure if `log_monitor` is a good permission name. What about `engin_view_log`?

Reply:
or `view_logs`

Reply:
We are using the same permission for studio logs and engine site logs. In that case `view_logs` would be more appropriate
@sumerjabri @avasquez614

Reply:
let's do `view_logs` for all then if that works.